includes/transformer/class-base.php

```diff
@@ -34,10 +39,10 @@ public static function transform( $object ) {
 	/**
 	 * Base constructor.
 	 *
-	 * @param stdClass $object
+	 * @param WP_Object|WP_Comment $wp_object
```

Comment:
```suggestion
	 * @param stdClass|WP_Post|WP_Comment $wp_object The WordPress object
```

Reply:
Do you already have use cases in mind where a non WP_Post or WP_Comment class is passed? Just curious! Of course I am fine with accepting that suggestion.

I know that for example 'The Events Calendar' offers it's own object with handy functions, but one could always create an instance of that within the transformer by the WP_Post object.

Reply:
I fear that not all WordPress plugins work with custom post types, but I am not sure... You are the expert there... We can remove `stdClass`.

Reply:
Some indeed use custom tables to handle complex functionality like recurring events, e.g. The Events Calendar. But that functionality is still built on top of a custom post type.

WordPress has been so clear for decades to use Custom Post Types wherever possible. I have not crossed any plugin that violated that suggestion.